src/services/bria/event-handler.ts

```diff
@@ -85,11 +85,17 @@ export const translate = (rawEvent: RawBriaEvent): BriaEvent | BriaEventError =>
   }
   const payoutInfo = rawAugmentation.getPayoutInfo()
   if (payoutInfo) {
+    const grpcMeta = payoutInfo.getMetadata()
     const info = payoutInfo.toObject()
+    let metadata: undefined | PayoutMetadata = undefined
+    if (grpcMeta) {
+      metadata = grpcMeta.toJavaScript()
```

Comment:
this is a really bad name (not our issue)... are we sure this returns a Json?

Reply:
Yes I'm sure this works (bats cronjob test runs through this code).